### ISSUE TITLE
Add term ectopic testis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10352]
**Preferred term label:**
Ectopic testis

**Synonyms**
Ectopic testicles

**Definition (free text, please give PubMed ID)**
The testis entering a location other than the scrotum after passing the inguinal canal. It usually derives from overdevelopment and lengthening of a segment of the gubernaculum or from obstruction of the entrance to the scrotum.

differing from undescended testis (cryptorchidism) in that ectopic testis is a congenitally abnormally located testis, that has descended from the abdominal cavity away from the normal path of descent while undescended testis are congenitally abnormally located testis within the normal path of descent.
PMID:31211064

**Parent term (use hpo.jax.org/app)**
Abnormal testis morphology HP:0000035

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: PMID:31211064

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000460
- label: Ectopic testis
- parents: [Abnormal testis morphology (HP:0000035)](https://hpo.jax.org/app/browse/term/HP:0000035)
- pmid: PMID:31211064
- def: Localization of the testis in an anatomic location other than the scrotum.
- comment: Ectopic testis usually derives from overdevelopment and lengthening of a segment of the gubernaculum or from obstruction of the entrance to the scrotum. differing from undescended testis (cryptorchidism) in that ectopic testis is a congenitally abnormally located testis, that has descended from the abdominal cavity away from the normal path of descent while undescended testis are congenitally abnormally located testis within the normal path of descent.
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10352
- synonyms:
